### PR TITLE
bug/65481 build agent image targets azure container registry

### PR DESCRIPTION
**Pull request by GuyHarwood**
- migrate from docker hub to azure container registry

### CODE REVIEW

File: deploy/build-agent/readme.md [hunk 1]
```diff
@@ -2,17 +2,17 @@
...
+- `create-instance.sh` creates a container instance using the latest image in your registry, via the Azure CLI.
 
 ## NOTE: Apple Silicon
 If you are building the image locally on an Apple Silicon device during developmet, you should use the `--platform linux/amd64` argument with the `docker build` command.  Otherwise, it will default to arm64, which lacks parity with the target hardware, and causes a lot of incompatibility issues during the image build.
 
 ## Steps
 1. update the `$AGENT_VERSION` variable in `./Dockerfile` to the [latest full release](https://github.com/microsoft/azure-pipelines-agent/releases)
 2. commit changes to ensure commit hash is aligned
-3. execute `./build-push-docker-hub.sh` to push new image
+3. execute `./build-push-registry.sh` to push new image
```
Comment (jon-shipley) [inline deploy/build-agent/readme.md:15]: Would you mind adding authentication steps to this guide? Is it `docker login`?

**Review by jon-shipley — APPROVED** [review]
LGTM :)

File: deploy/build-agent/readme.md (hunk 1 above)
Comment (GuyHarwood) [reply deploy/build-agent/readme.md]: updated

**Comment by GuyHarwood** [comment]
reason found - https://github.com/Azure/acr/discussions/710

**Pull request merged**